src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/sentencing/SentencingResource.kt

```diff
@@ -314,6 +314,89 @@ class SentencingResource(private val sentencingService: SentencingService) {
     request: CreateCourtCaseRequest,
   ): CreateCourtCaseResponse =
     sentencingService.createCourtCase(offenderNo, request)
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_SENTENCING')")
+  @PostMapping("/prisoners/{offenderNo}/sentencing/court-cases/{caseId}")
```

Comment:
minor - but I think a URL of `/prisoners/{offenderNo}/sentencing/court-cases/{caseId}/court-appearance` probably reads better

Reply:
agreed 👍